javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,9 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  var arr = [];
+  for (var i = 0; i < words.length; i++) {
+    words[i] += suffix;
```

Comment:
This will work, but it does end up changing the original array, which is a no-no. Unless it is the purpose of the function, functions should work with the data from objects and arrays, but not change the original.